autoload/neoterm.vim

```diff
@@ -17,11 +18,25 @@ function! neoterm#exec(list)
     end
 
     exec split_cmd | exec current_window . "wincmd w | set noim"
+  elseif exists('g:neoterm_buffer_id') && bufwinnr(g:neoterm_buffer_id) == -1
```

Comment:
The elseif checks neoterm_buffer_id and not neoterm_current_id, so technically its not really implied.